package.json

```diff
@@ -0,0 +1,574 @@
+{
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.26.0",
+    "hint": "^7.1.2",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  },
+  "name": "hidar37.github.io",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "version": "1.0.0",
+  "main": "index.js",
+  "dependencies": {
+    "abab": "^2.0.6",
+    "abbrev": "^1.1.1",
+    "acorn": "^8.8.0",
+    "acorn-globals": "^6.0.0",
```

Comment:
- [ ] I think you have something wrong with your package.json file so kindly fix it to make your linters works fine because I can see you disables all the linters so try to delete node_modules and run `npm install` again to install again the packages for you 📌💯🚀